README.md

```diff
@@ -1,7 +1,114 @@
-# Tour Savvy App
+# Tour Savvy App Frontend
```

Comment:
- [ ] In the "add package" section:

![image](https://github.com/Zafron047/tour_savvy_frontend/assets/57421823/ba244f2d-2729-4cca-945c-e1a1e9a51327)

This isn't very clear.  One says "Add" and the other "Create".  What button should the user push?  Kindly rename the buttons or add clarification. 👍

Reply:
@Meltrust 
We have worked on removing the confusion while add and create seems same from the user endpoint.

![addpackage](https://github.com/Zafron047/tour_savvy_frontend/assets/117600135/c2f78e41-ed8b-4f79-9b29-0370bc51335a)

Replaced create with a submit button. 😄 

Thanks for the feedback.